statsmodels/multivariate/factor.py

```diff
@@ -433,12 +433,15 @@ def nscore(par):
 
         return FactorResults(self)
 
-    def _fit_ml_em(self, iter):
+    def _fit_ml_em(self, iter, random_state=None):
         """estimate Factor model using EM algorithm
         """
         # Starting values
-        np.random.seed(3427)
-        load = 0.1*np.random.normal(size=(self.k_endog, self.n_factor))
+        if random_state is None:
+            rs = np.random.RandomState(3427)
+        else:
+            rs = random_state
+        load = 0.1 * rs.normal(size=(self.k_endog, self.n_factor))
```

Comment:
Could you switch to standard_normal?  It is faster than normal.

Reply:
Yep! Just did it.